src/components/Main/Main.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './Main.scss';
+import { Route, Switch } from 'react-router';
+import Home from '../../pages/Home';
+import AdminOrders from '../../pages/AdminOrders';
+import AdminProducts from '../../pages/AdminProducts';
+import CheckOut from '../../pages/CheckOut';
+import Orders from '../../pages/Orders';
+import OrderSuccess from '../../pages/OrderSuccess';
+import Products from '../../pages/Products';
+import ShoppingCart from '../../pages/ShoppingCart';
+
+function Main() {
+  return (
+    <main className="main">
+      <Switch>
+        <Route exact path="/" components={Home}></Route>
+        <Route exact path="/AdminOrders" components={AdminOrders}></Route>
+        <Route exact path="/AdminProducts" components={AdminProducts}></Route>
+        <Route exact path="/CheckOut" components={CheckOut}></Route>
+        <Route exact path="/Orders" components={Orders}></Route>
+        <Route exact path="/OrderSuccess" components={OrderSuccess}></Route>
```

Comment:
Let's try adding a path that re-directs us to '/' if they type in something that doesn't match any of the options in the url. There are multiple ways to do this so when you find a solution you like reach out so we can discuss first before you implement it.